src/components/layout/AddButton.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+	<v-btn
+		class="base-main_add-button"
+		fixed
+		dark
+		fab
+		bottom
+		right
+		:no-resize="true"
+		color="green"
+		@click="createNewMessage()"
+	>
+		<i class="material-icons">
+			create
+		</i>
+	</v-btn>
+</template>
+
+<script>
+	export default {
+		name: "AddButton",
+		methods: {
+			createNewMessage() {
```

Comment:
Why simple reusable button have any methods? U need to add this method to place where you use it

Reply:
i don't understand what you mean